src/current/_includes/releases/v24.2/v24.2.0.md

```diff
@@ -0,0 +1,144 @@
+
+- **Feature categories**
+    - [Change data capture](#v24-2-0-change-data-capture)
+    - [Migrations](#v24-2-0-migrations)
+    - [SQL](#v24-2-0-sql)
+    - [Operations](#v24-2-0-operations)
+    - [Observability](#v24-2-0-observability)
+
+- **Additional information**
+    - [Backward-incompatible changes and deprecations](#v24-2-0-backward-incompatible-changes-and-deprecations)
+    - [Key cluster setting changes](#v24-2-0-key-cluster-setting-changes)
+    - [Known limitations](#v24-2-0-known-limitations)
+    - [Additional resources](#v24-2-0-additional-resources)
+
+{{ site.data.alerts.callout_info }}
+In CockroachDB Self-Hosted, all available features are free to use unless their description specifies that an Enterprise license is required. For more information, refer to the [Licensing FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs).
+{{ site.data.alerts.end }}
+
+<div id="feature-highlights">
+
+<h4 id="v24-2-0-change-data-capture">Change Data Capture</h4>
+
+{% include releases/v24.2/feature-highlights-change-data-capture.html %}
+
+<h4 id="v24-2-0-migrations">Migrations</h4>
+
+{% include releases/v24.2/feature-highlights-migrations.html %}
+
+<h4 id="v24-2-0-sql">SQL</h4>
+
+{% include releases/v24.2/feature-highlights-sql.html %}
+
+<h4 id="v24-2-0-operations">Operations</h4>
+
+{% include releases/v24.2/feature-highlights-operations.html %}
+
+<h4 id="v24-2-0-observability">Observability</h4>
+
+{% include releases/v24.2/feature-highlights-observability.html %}
+
+{% include releases/v24.2/feature-detail-key.html %}
+
+</div>
+
+<h4 id="v24-2-0-backward-incompatible-changes-and-deprecations">Backward-incompatible changes and deprecations</h4>
+
+CockroachDB v24.2.0 includes no backward-incompatible changes or deprecations.
+
+<h4 id="v24-2-0-key-cluster-setting-changes">Key Cluster Setting Changes</h4>
+
+The following changes should be reviewed prior to upgrading. Default cluster settings will be used unless you have manually set a value for a setting. This can be confirmed by running the SQL statement `SELECT * FROM system.settings` to view the non-default settings.
+
+- [Settings added](#v24-2-0-settings-added)
+- [Settings removed](#v24-2-0-settings-removed)
+- [Settings with changed defaults](#v24-2-0-settings-with-changed-defaults)
+- [Settings with new options](#v24-2-0-settings-with-new-options)
+- [Settings with new aliases](#v24-2-0-settings-with-new-aliases)
+
+<h5 id="v24-2-0-settings-added">Settings added</h5>
+
+- `debug.zip.redact_addresses`: Added the [cluster setting]({% link v24.2/cluster-settings.md %}) `debug.zip.redact_addresses.enabled` that allows the user to enable or disable redaction of fields like `hostname` and IP addresses. [#123544][#123544] |
```

Comment:
I haven't noticed these before, but it seems redundant to have the cluster setting mentioned as the key for the list item and then a few words later, mentioned again with a link. Also, it's a little strange to consider the setting _value_ to be part of the setting name (key).

Reply:
will mark this as resolved but agree that this should be improved going forward